### PR TITLE
English translations update

### PR DESCRIPTION
**Pull request by laanwj**
Update for Transifex. Needed after bitcoin/bitcoin#21836.

### CODE REVIEW

File: src/qt/bitcoinstrings.cpp [hunk 1]
```diff
@@ -10,239 +10,4 @@
 #endif
 static const char UNUSED *bitcoin_strings[] = {
 QT_TRANSLATE_NOOP("bitcoin-core", "The %s developers"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
-"%s corrupt. Try using the wallet tool bitcoin-wallet to salvage or restoring "
-"a backup."),
```
[the remark this answers is not in the document]
Comment (hebasto) [reply src/qt/bitcoinstrings.cpp:15]: Yeah, the same.

This diff fixes it:
```diff
--- a/share/qt/extract_strings_qt.py
+++ b/share/qt/extract_strings_qt.py
@@ -58,7 +58,7 @@ if not XGETTEXT:
     print('Cannot extract strings: xgettext utility is not installed or not configured.',file=sys.stderr)
     print('Please install package "gettext" and re-run \'./configure\'.',file=sys.stderr)
     sys.exit(1)
-child = Popen([XGETTEXT,'--output=-','-n','--keyword=_'] + files, stdout=PIPE)
+child = Popen([XGETTEXT,'--output=-','--from-code=utf-8','-n','--keyword=_'] + files, stdout=PIPE)
 (out, err) = child.communicate()
 
 messages = parse_po(out.decode('utf-8'))
```

Details: https://bugzilla.redhat.com/show_bug.cgi?id=237786

**Comment by hebasto** [comment]
@laanwj 

If this PR will be updated, could you rebase it to include the recent changes from #296?

**Comment by laanwj** [comment]
Updated.

(I only just noticed I made this branch to the gui repository itself instead of my fork. Sorry for that.)

**Review by hebasto — APPROVED** [review]
ACK c34a49f77f474c94e005331458111308a19afd53, verified that `make -C src translate` returns zero updated strings (as all of them are updated in the top commit):
```
...
Updating 'qt/locale/bitcoin_en.ts'...
    Found 1048 source text(s) (0 new and 1048 already existing)
...
```

**Pull request merged**